omod/src/main/webapp/fragments/field/multipleInputDate.gsp

```diff
@@ -101,7 +101,7 @@
 
 <p id="${config.id}">
     <label for="${ config.id }-field">
-        ${ config.label } <% if (config.classes && config.classes.contains("required")) { %><span>(${ ui.message("emr.formValidation.messages.requiredField.label") })</span><% } %>
+        ${ config.label } <% if (config.classes && config.classes.contains("requiredTitle")) { %><span>(${ ui.message("emr.formValidation.messages.requiredField.label") })</span><% } %>
```

Comment:
Why this change from required to requiredTitle?

Reply:
I responded in related PR - https://github.com/openmrs/openmrs-module-registrationapp/pull/78/